### PR TITLE
feat(ado-improvement-1): Remove PR Comments, add output to logs

### PR DESCRIPTION
**Pull request by brocktaylor7**
#### Details

This removes PR comments from both the Github Action and the ADO Extension and moves the output to the logs. This output to the logs will output every time the extension runs, rather than only on PR runs.

Sample log output for Github:
```
Run echo 'Starting accessibility scanner...'
Starting accessibility scanner...
* **URLs**: 2 URL(s) failed, 2 URL(s) passed, and 0 were not scannable
* **Rules**: 6 check(s) failed, 11 check(s) passed, and 38 were not applicable
* Download the **Accessibility Insights artifact** to view the detailed results of these checks
#### Failed instances
* **3 × label**:  Ensures every form element has a label
* **2 × html-has-lang**:  Ensures every HTML document has a lang attribute
* **1 × image-alt**:  Ensures \<img> elements have alternate text or a role of none or presentation

---
This scan used [axe-core 4.3.2](https://github.com/dequelabs/axe-core/releases/tag/v4.3.2) with Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.75 Safari/537.36.
```

Sample log output for ADO:
```
##[debug] ### Results from Current Run
### ![Accessibility Insights](https://accessibilityinsights.io/img/a11yinsights-blue.svg) Accessibility Insights Action

**22 failure instances**
* (13) **label**:  Ensures every form element has a label
* (4) **color-contrast**:  Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds
* (4) **image-alt**:  Ensures \<img> elements have alternate text or a role of none or presentation
* (1) **html-has-lang**:  Ensures every HTML document has a lang attribute

**Baseline not configured**
A baseline lets you mark known failures so it's easier to identify new failures as they're introduced. See [baselining docs](https://github.com/microsoft/accessibility-insights-action/blob/main/docs/ado-extension-usage.md#using-a-baseline-file) for more.

See all failures and scan details by downloading the report from [run artifacts](undefined)

---
#### Scan summary
**URLs**: 1 with failures, 0 passed, 0 not scannable
**Rules**: 22 with failures, 11 passed, 37 not applicable

---
This scan used [axe-core 4.3.2](https://github.com/dequelabs/axe-core/releases/tag/v4.3.2) with Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.75 Safari/537.36.
```

##### Motivation

Feature work.

##### Context

We will better format the output for consumption via log files in a future PR.

#### Pull request checklist

- [n/a] Addresses an existing issue: Fixes #0000
- [x] Added relevant unit test for your changes. (`yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [ ] (after PR created) The `Accessibility Checks (pull_request)` check should fail. All other checks should pass.

### CODE REVIEW

File: packages/gh-action/src/console/console-comment-creator.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    public start(): Promise<void> {
+        // We don't do anything for pull request flow
+        this.logger.logVerbose('console comment creator started');
+        return Promise.resolve();
+    }
+
+    public async completeRun(combinedReportResult: CombinedReportParameters): Promise<void> {
+        const reportMarkdown = this.reportMarkdownConvertor.convert(combinedReportResult);
+        this.logMessage(reportMarkdown);
```
Comment (DaveTryon) [inline packages/gh-action/src/console/console-comment-creator.ts:25]: Does this play nicely with the stdoutTransformer?

**Comment by DaveTryon** [comment]
Is the sample log from ADO or GitHub? Would it be possible to provide both?

File: packages/gh-action/src/console/console-comment-creator.ts (hunk 1 above)
Comment (brocktaylor7) [reply packages/gh-action/src/console/console-comment-creator.ts:25]: It does, it's simply a wrapper around Logger.LogInfo, and outputs with the standard [Trace][info] tag coming from the ConsoleLoggerClient.

Comment (brocktaylor7) [reply packages/gh-action/src/console/console-comment-creator.ts]: That was a leftover from the previous implementation of the code. That particular abstraction isn't really helpful, especially now where we're doing more sophisticated things with the logging, so I've removed it for clarity's sake.

**Comment by brocktaylor7** [comment]
> Is the sample log from ADO or GitHub? Would it be possible to provide both?

This is a good call, I've added the ADO comment sample to the description. I came back to this with fresh eyes and did a bit of cleanup as well.

**Review by DaveTryon — APPROVED** [review]
Thanks for clarifying that the markdown won't be the final format in the log file. That would be a really odd combination 🤢

**Pull request merged**